packages/plugin-odoo/src/Odoo.ts

```diff
@@ -127,6 +197,68 @@ export class Odoo<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
                     return [button];
                 },
             },
+            this._getPaddingComponent('OdooImagePaddingNoneActionable', OdooPaddingClasses.NONE),
+            this._getPaddingComponent('OdooImagePaddingSmallActionable', OdooPaddingClasses.SMALL),
+            this._getPaddingComponent(
+                'OdooImagePaddingMediumActionable',
+                OdooPaddingClasses.MEDIUM,
+            ),
+            this._getPaddingComponent('OdooImagePaddingLargeActionable', OdooPaddingClasses.LARGE),
+            this._getPaddingComponent('OdooImagePaddingXLActionable', OdooPaddingClasses.XL),
+
+            this._getImageWidthComponent('OdooImageWidthAutoActionable', 'auto'),
+            this._getImageWidthComponent('OdooImageWidth25Actionable', '25'),
+            this._getImageWidthComponent('OdooImageWidth50Actionable', '50'),
+            this._getImageWidthComponent('OdooImageWidth75Actionable', '75'),
+            this._getImageWidthComponent('OdooImageWidth100Actionable', '100'),
+
+            this._getImageClassComponent(
+                'OdooImageRoundedActionable',
+                OdooImageClasses.ROUNDED,
+                'fa-square',
+            ),
+            this._getImageClassComponent(
+                'OdooImageRoundedCircleActionable',
+                OdooImageClasses.ROUNDED_CIRCLE,
+                'fa-circle-o',
+            ),
+            this._getImageClassComponent(
+                'OdooImageRoundedShadowActionable',
+                OdooImageClasses.SHADOW,
+                'fa-sun-o',
+            ),
+            this._getImageClassComponent(
+                'OdooImageRoundedThumbnailActionable',
+                OdooImageClasses.IMG_THUMBNAIL,
+                'fa-picture-o',
+            ),
```

Comment:
Wow t'as fais tous ces trucs pourris là d'Odoo d'un coup ? Beau boulot. 👍  (Je les aurais préfixé de `make` ou `create` plutôt que `get` mais si tu tiens à tes `get` je ne vais pas me battre c'est pas "invalide".)

Reply:
`make` is also a great prefix. I changed it.